### ISSUE TITLE
[Question] Support for Custom Attention Mask

### ISSUE THREAD

**Issue by Peng-YM** [mlc-ai/mlc-llm#2232]
## ❓ General Questions

Many thanks for your effort to develop such a great library. I want to add support for the ChatGLM (not the 3rd generation) model in mlc-llm, however, it seems to me that currently attention mask is not supported as documented in the following file:

```python
# 3rdparty/tvm/python/tvm/relax/frontend/nn/modules.py:924

assert attention_mask is None, "Attention mask not yet supported."
``` 

How can I resolve this issue, sincerely wish to hear from you.

**Comment by onedayherocoming** [comment]
https://github.com/flashinfer-ai/flashinfer/pull/266
FlashInfer has supported custom mask, any plan to support custom attention mask ?
